### ISSUE TITLE
[BUG] - Predictions in FMS go away

### ISSUE THREAD

**Issue by dr71205** [invernyx/md11-bugs#1420]
### Describe the bug

Predictions in FMS go away

### Steps to reproduce

Honestly I don't know what happened, although I do know that there was for some reason a gap in routeing between 5820N and 5830N
The route was imported through Simbrief

### Expected behaviour

Predictions work and there is no gap in routeing

### Evidence

![image](https://github.com/invernyx/md11-bugs/assets/150094502/4e32cd57-3acb-4325-a7fe-3776b61fa03e)

[PREDBUG.json](https://github.com/user-attachments/files/15765943/PREDBUG.json)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Phase of flight

Cruise

### Altitude

FL320

### Departure

EHAM

### Arrival

MMMX

### Cruise level

FL320

### ZFW

179.0 MT

### CG

24.1%

### Fuel

85200kgs

### Cost index

100

### Route

EHAM/24 N0491F300 BERG2S BERGI L602 SUPUR P1 ROKAN N96 EVTAR DCT ERAKA/M083F300 DCT 58N020W 58N030W 58N040W 56N050W DCT JANJO/N0490F300 N714B YBC/N0488F340 DCT YXI DCT YXU/N0488F360 DCT ROD DCT IIU Q139 HITMN DCT MERDN DCT MCB DCT TBD M345 AXEXO UM345 PAZ UT154 ENAGA ENAG2A MMMX/05R

### Location

250 NM away from 5830N, inbound bearing 289

### Agreement

- [X] Yes

**Comment by dr71205** [comment]
![image](https://github.com/invernyx/md11-bugs/assets/150094502/94f1864d-6e33-499d-b135-2a2610ef17e5)
Between 5830N and COAPA predictions are gone, but from PAZ right around TOD they come back in

**Comment by dr71205** [comment]
The workarounds of reinserting CI or directing to next waypoint did not work, until passing 5840N, at which point DCT 5650N returned predictions

**Comment by dr71205** [comment]
Changing CRZ level in the INIT page seems to break it

**Comment by dr71205** [comment]
340/360 _does_ work, however, leading me to believe something is amiss with step climb logic

**Comment by dr71205** [comment]
turns out that's not the issue, but it seems to occur _during_ the stepclimb

**Comment by dr71205** [comment]
Aside from the root cause of CRZ level changes breaking predictions, any other causes for it not being recoverable just seem to be random. Climbing from 340 to 360 with 360/380 led to it being unrecoverable, even after reaching FL360, but then entering 360/380/400 in the INIT page seemingly led to it being recoverable again.

TL;DR, root cause of losing predictions seems to be either taking a direct or recruising, but as for it being unrecoverable might be correlation rather than causation

**Comment by turbofandude** [comment]
Please retry and re-submit with 0.62+. We fixed several issues surrounding this system.
